runtests/runtests.js

```diff
@@ -325,7 +325,7 @@ function executeTest(options, callback) {
         args.push('--output', tempInput)
         args.push('--prologue', tempPrologue)
 
-        child_process.execFile('python', args, {}, compileDone)
+        child_process.execFile('python2.7', args, {}, compileDone)
```

Comment:
There are unfortunately some differences between various python environments. Other parts of the codebase detect the proper name; for example, Makefile uses:
```
PYTHON := $(shell { command -v python2 || command -v python; } 2>/dev/null)
```

I'd take this diff out of the pull and figure it out in a separate pull.

Reply:
See #2239 where I moved this diff.